BUG: fresh install results in ImportError / Pandera AttributeError

A fresh install + import of crowsetta right now gives the following error

```console
ImportError while loading conftest '/Users/davidnicholson/Documents/repos/vocalpy/crowsetta/tests/conftest.py'.
tests/__init__.py:1: in <module>
    from .fixtures import *
tests/fixtures/__init__.py:3: in <module>
    from .bbox import *
tests/fixtures/bbox.py:4: in <module>
    import crowsetta
.nox/test-3-10/lib/python3.10/site-packages/crowsetta/__init__.py:21: in <module>
    from . import formats
.nox/test-3-10/lib/python3.10/site-packages/crowsetta/formats/__init__.py:7: in <module>
    from . import bbox, seq
.nox/test-3-10/lib/python3.10/site-packages/crowsetta/formats/bbox/__init__.py:1: in <module>
    from .audbbox import AudBBox
.nox/test-3-10/lib/python3.10/site-packages/crowsetta/formats/bbox/audbbox.py:99: in <module>
    class AudBBoxSchema(pandera.SchemaModel):
E   AttributeError: module 'pandera' has no attribute 'SchemaModel'
```

I think what happened is that SchemaModel got replaced with DataFrameModel in [pandera 0.20.0](https://github.com/unionai-oss/pandera/releases/tag/v0.20.0) so the fix is just to rename SchemaModel throughout

Ralph Peterson reported a related issue with installing vocalpy, where crowsetta threw an ImportError, because of a pandas version that didn't work with numpy 2.0. But I think that might actually have been fixed by pandas/pandera releases in the meantime because I'm not getting the error

https://github.com/vocalpy/vocalpy/issues/173

--- a/src/crowsetta/formats/bbox/audbbox.py
+++ b/src/crowsetta/formats/bbox/audbbox.py
@@ -96,8 +96,12 @@ def df_to_lines(df: pd.DataFrame) -> list[str]:
     return lines
 
 
-class AudBBoxSchema(pandera.SchemaModel):
-    """A :class:`pandera.SchemaModel` that
+class AudBBoxSchema(pandera.DataFrameModel
+
+):
+    """A :class:`pandera.DataFrameModel
+
+` that
     validates :mod:`pandas` dataframes
     loaded from Audacity label tracks
     in extended format, exported to txt files

--- a/src/crowsetta/formats/bbox/raven.py
+++ b/src/crowsetta/formats/bbox/raven.py
@@ -17,8 +17,12 @@ import crowsetta
 from crowsetta.typing import PathLike
 
 
-class RavenSchema(pandera.SchemaModel):
-    """A :class:`pandera.SchemaModel` that validates :type:`pandas.DataFrame`s
+class RavenSchema(pandera.DataFrameModel
+
+):
+    """A :class:`pandera.DataFrameModel
+
+` that validates :type:`pandas.DataFrame`s
     loaded from a txt file, created by exporting a Selection Table
     from Raven.
     """

--- a/src/crowsetta/formats/seq/audseq.py
+++ b/src/crowsetta/formats/seq/audseq.py
@@ -15,8 +15,12 @@ import crowsetta
 from crowsetta.typing import PathLike
 
 
-class AudSeqSchema(pandera.SchemaModel):
-    """A :class:`pandera.SchemaModel`
+class AudSeqSchema(pandera.DataFrameModel
+
+):
+    """A :class:`pandera.DataFrameModel
+
+`
     that validates :type:`pandas.DataFrame`s
     loaded from Audacity Labeltrack annotations
     exported to txt files in the standard format.

--- a/src/crowsetta/formats/seq/generic.py
+++ b/src/crowsetta/formats/seq/generic.py
@@ -35,8 +35,12 @@ Both units can also be specified. Conversion between units is not validated.
 """
 
 
-class GenericSeqSchema(pandera.SchemaModel):
-    """A :class: `pandera.SchemaModel` that validates
+class GenericSeqSchema(pandera.DataFrameModel
+
+):
+    """A :class: `pandera.DataFrameModel
+
+` that validates
     :type:`pandas.DataFrame`s
     loaded from a csv file  in the ``'generic-seq'`` annotation
     format.

--- a/src/crowsetta/formats/seq/simple.py
+++ b/src/crowsetta/formats/seq/simple.py
@@ -30,8 +30,12 @@ import crowsetta
 from crowsetta.typing import PathLike
 
 
-class SimpleSeqSchema(pandera.SchemaModel):
-    """A :class:`pandera.SchemaModel`
+class SimpleSeqSchema(pandera.DataFrameModel
+
+):
+    """A :class:`pandera.DataFrameModel
+
+`
     that validates :type:`pandas.DataFrame`s
     loaded from a csv or txt file in a 'simple-seq' format.
 

--- a/src/crowsetta/formats/seq/timit.py
+++ b/src/crowsetta/formats/seq/timit.py
@@ -19,8 +19,12 @@ import crowsetta
 from crowsetta.typing import PathLike
 
 
-class TimitTranscriptSchema(pandera.SchemaModel):
-    """A :class:`pandera.SchemaModel` that validates :type:`pandas.DataFrame`s
+class TimitTranscriptSchema(pandera.DataFrameModel
+
+):
+    """A :class:`pandera.DataFrameModel
+
+` that validates :type:`pandas.DataFrame`s
     loaded from a phn or wrd file in the TIMIT[1]_ transcription format.
 
     References

